packages/fxa-auth-server/package.json

```diff
@@ -24,7 +24,7 @@
     "test-remote": "MAILER_HOST=restmail.net MAILER_PORT=80 CORS_ORIGIN=http://baz mocha -r ts-node/register --timeout=300000 test/remote",
     "format": "prettier --write --config ../../_dev/.prettierrc '**'",
     "gen-keys": "ts-node ./scripts/gen_keys.js; ts-node ./scripts/oauth_gen_keys.js; ts-node ./scripts/gen_vapid_keys.js",
-    "write-emails": "node -r ts-node/register ./scripts/write-emails-to-disk.js",
+    "write-emails": "yarn emails-scss && node -r ts-node/register ./scripts/write-emails-to-disk.js",
```

Comment:
Related to this comment: https://github.com/mozilla/fxa/pull/9923/files#r677542177

I thought `yarn write-emails` was for the old templates? When I try to run this, I get:

![image](https://user-images.githubusercontent.com/13018240/127179792-96e38fdf-82f7-4605-9e75-4acd06cb771e.png)

Is there another step, or are we no longer using `write-templates`?

I think we also want to take use of `yarn` out and use `npm run` instead.

On a related note, CI is failing with this (not sure if you can see):
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/13018240/127180633-e9b9528c-2bff-4230-b952-a93b1f32b1e8.png">